packages/eomaps.yml

```
@@ -0,0 +1,8 @@
+repo: raphaelquast/eomaps
+section: mapping
+description: A library to create interactive maps of geographical datasets
+site: https://github.com/raphaelquast/EOmaps
```

```suggestion
site: https://raphaelquast.github.io/EOmaps/ 
```
?  We already link the repo...